package.json

```diff
@@ -34,27 +37,21 @@
         "schema": "node scripts/generate-schemas.mjs",
         "predeploy": "yarn build-editor; echo \"gosling.js.org\" >> build/CNAME",
         "deploy": "gh-pages -d build",
-        "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md"
+        "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+        "knip": "knip --config knip.config.json"
```

Comment:
This is a nice addition! I see many unused components as an output :)

Reply:
Trevor suggested the tool! Seems quite useful for this type of cleanup.